src/level_four/core/tasks/send_transaction_tasks_eth.c

```diff
@@ -233,7 +233,7 @@ void send_transaction_tasks_eth()
 
         instruction_scr_destructor();
         eth_get_fee_string(&eth_unsigned_txn_ptr, fee, sizeof(fee),
-                           var_send_transaction_data.transaction_metadata.decimal[0]);
```

Comment:
Let's change the name of `decimal` variable to `eth_val_decimal` to remove this confusion

Reply:
Changes added